test/e2eclusterctl/data/infrastructure-capz/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
```

Comment:
IMO this file can be omitted because it provides the same settings already existing in embedded metadata at  https://github.com/kubernetes-sigs/cluster-api/blob/8b29a9adeeba4677da4221b887cce61155f925a3/cmd/clusterctl/client/repository/metadata_client.go#L189